src/layouts/SatelliteToken.tsx

```diff
@@ -38,11 +47,6 @@ const SatelliteToken: React.FC = () => {
       <Page
         className="chr-c-all-services"
         onPageResize={null} // required to disable PF resize observer that causes re-rendring issue
-        header={
-          <Masthead className="chr-c-masthead">
```

Comment:
@Hyperkid123 since we are now exposing this as a module from chrome at `/insights/satellite` - this is causing the insights masthead and nav to render around the module (hence removing the masthead here otherwise we see 2).

Reply:
Does this page require some unique layout?

Reply:
No - the content is just a bit odd/out of the ordinary as it is 3 utilities.